Description: folder with static web pages are moved to /usr/share/moin/htdocs
 only applicable on Debian systems storing web pages system-wide
Author: Frank Lin PIAT <fpiat@klabs.be>
Forwarded: not-needed
Last-Update: 2009-11-15
--- a/MoinMoin/web/static/__init__.py
+++ b/MoinMoin/web/static/__init__.py
@@ -52,7 +52,7 @@
 
 from werkzeug import SharedDataMiddleware
 
-STATIC_FILES_PATH = join(abspath(dirname(__file__)), 'htdocs')
+STATIC_FILES_PATH = '/usr/share/moin/htdocs'
 
 
 def make_static_serving_app(application, shared):
